### ISSUE TITLE
Support for SSH based plugins?

### ISSUE THREAD

**Issue by callumj** [buildkite-plugins/buildkite-plugin-linter#208]
We have our plugin example that pulls from SSH instead of HTTPS and the linter currently doesn't like this example in the README

```yml
steps:
  - plugins:
      - ssh://git@github.com/hoteltonight/jenkins-trigger-buildkite-plugin#v0.2:
          job_name: hotelstonight-docker-build
          job_token: hackme
    branches: master
```

**Comment by toolmantim** [comment]
Ah yes! I'll get this fixed. What id are you specifying for the plugin? `hoteltonight/jenkins-trigger`?

**Comment by callumj** [comment]
Hey @toolmantim, that's correct.

**Comment by toolmantim** [comment]
This was just fixed in #219 @callumj!

Can you update the `id` to `"ssh://git@github.com/hoteltonight/jenkins-trigger-buildkite-plugin"` and give it another go?
